examples/Example/src/draggable/index.tsx

```diff
@@ -43,7 +53,8 @@ export class DraggableBox extends Component {
       <PanGestureHandler
         {...this.props}
         onGestureEvent={this._onGestureEvent}
-        onHandlerStateChange={this._onHandlerStateChange}>
+        onHandlerStateChange={this._onHandlerStateChange}
+        minDist={this.props.minDist}>
```

Comment:
I added this since in one of the examples, `minDist` is passed to `DraggableBox` in one example.